helios-tools/src/main/java/com/spotify/helios/cli/command/WildcardJobCommand.java

```diff
@@ -81,13 +81,13 @@ int run(final Namespace options, final HeliosClient client, final PrintStream ou
       return 1;
     }
 
-    final JobId jobId = Iterables.getOnlyElement(jobs.keySet());
+    final Job job = Iterables.getOnlyElement(jobs.values());
 
-    return runWithJobId(options, client, out, json, jobId, stdin);
+    return runWithJobId(options, client, out, json, job, stdin);
   }
 
   protected abstract int runWithJobId(final Namespace options, final HeliosClient client,
```

Comment:
should probably rename this method now since it refers to `JobId`

Reply:
good catch. done